com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -168,10 +171,14 @@ public UniversalRenderPipeline(UniversalRenderPipelineAsset asset)
             CameraCaptureBridge.enabled = true;
 
             RenderingUtils.ClearSystemInfoCache();
+
+            m_DebugDisplaySettingsUI.RegisterDebug(DebugDisplaySettings.Instance);
```

Comment:
IIRC we decided we will land first a version with the debug disable right? If so shouldn't this be guarded by a conditional compile directive?

Reply:
Yep, this is now behind URP_ENABLE_DEBUG_VIEWS switch.